src/main/java/nl/tudelft/pixelperfect/gamestates/PlayState.java

```diff
@@ -76,7 +76,7 @@ public void updateMovement(float tpf) {
       observer.rotate(0, -0.75f * tpf, 0);
     }
     if (game.isDebugTrigger()) {
```

Comment:
Might want to hold off on this as we dont want to use a keyboard in the end anyway, But this can be done by using KeyListner instead of actionListner